qiskit/circuit/reset.py

```diff
@@ -18,10 +18,13 @@
 
 
 class Reset(SingletonInstruction):
-    """Qubit reset."""
+    r"""Incoherently reset a qubit to the :math:`\lvert0\rangle` state."""
```

Comment:
I'm feeling the word _incoherently_ might add more confusion than not...

Reply:
It's standard quantum-computing terminology, but if you have a preferred word, I can change it.